mycore-base/src/main/java/org/mycore/common/MCRCalendar.java

```diff
@@ -651,76 +651,58 @@ protected static GregorianCalendar getCalendarFromPersicDate(String datestr, boo
             final boolean bb = beforeZero(dateTrimmed, CalendarType.Persic);
             final String cleanDate = cleanDate(dateTrimmed, CalendarType.Persic);
             final int[] fields = parseDateString(cleanDate, last, CalendarType.Persic);
-            final int year = fields[0];
+            int year = fields[0];
```

Comment:
Same as:
`final year = bb ? 622 - fields[0] : 621 + fields[0] ;`
Or?

Reply:
is addressed in https://github.com/MyCoRe-Org/mycore/pull/2050/commits/a80138fbebbc50ad293b2d31a6649aa63cc0d35c